airgun/entities/lifecycleenvironment.py

```diff
@@ -0,0 +1,85 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.lifecycleenvironment import (
+    LCECreateView,
+    LCEEditView,
+    LCEView,
+)
+
+
+class LCEEntity(BaseEntity):
+
+    def create_environment_path(self, values):
+        view = self.navigate_to(self, 'New Path')
+        view.fill(values)
+        view.submit.click()
+
+    def create_environment(self, entity_name, values):
+        view = self.navigate_to(
+            self, 'New Environment', entity_name=entity_name)
+        view.fill(values)
+        view.submit.click()
+
+    def create(self, prior_entity_name=None, values=None):
```

Comment:
Should be `def create(self, values, prior_entity_name=None)` - values are required and should be specified in any case. Also such order will allow you skipping explicit specification of `.create(value={...})` when `prior_entity_name` is not needed.